x-pack/plugins/aiops/public/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AiopsPlugin } from './plugin';
+
+/**
+ * aiops plugin public setup contract
+ */
+export type AiopsPluginSetup = ReturnType<AiopsPlugin['setup']>;
+
+/**
+ * aiops plugin publis start contract
```

Comment:
Typo! `public` instead of `publis`

Reply:
Fixed in [1c63220](https://github.com/elastic/kibana/pull/131317/commits/1c63220f3117c8edbf0ee7afecf7a4dd4d733f22).